src/utils/shuffleArray.ts

```diff
@@ -0,0 +1,9 @@
+export const shuffleArray = (array: any[]): any[] => {
```

Comment:
```suggestion
export function shuffleArray<T>(array: T): T {
```

Use inferring instead of any please